lib/containers/common.pm

```diff
@@ -254,7 +254,8 @@ sub test_container_image {
     # Images from custom registry are listed with the '$registry/library/'
     $image =~ s/^docker\.io\/library\///;
 
-    my $smoketest = qq[/bin/sh -c '/bin/uname -r; /bin/echo "Heartbeat from $image"; ps'];
+    my $wait = qq[ ; echo Sync 10 sec; sleep 10];
+    my $smoketest = qq[/bin/sh -c '/bin/uname -r; /bin/echo "Heartbeat from $image"; ps'] . $wait;
```

Comment:
Is this really needed? Sleeping in tests is in general bad practice and should be avoided whenever possible.

Reply:
I fail to see how an additional `sleep` in the command to be executed should resolve a timeout in `podman container create`. Keep in mind, `container create` does not execute the container, it only creates it. This argument should not make any difference there.

Reply:
@m-dati could you clone the original task without your changes to check, if the test passes on marvin also without those changes? I have the feeling the issue lies somewhere else.

Reply:
The error is not in the `container create` command itself, but in the openqa expected mark string missing in the stdout, so that the timeout is triggered.
There is some strange delay of the create-command that let it still work in delay after exit so that it overwrites the echo xxxx-0- of expected end-command.
The short sleep resolves that.

Reply:
> The error is not in the `container create` command itself, but in the openqa expected mark string missing in the stdout, so that the timeout is triggered. There is some strange delay of the create-command that let it still work in delay after exit so that it overwrites the echo xxxx-0- of expected end-command. The short sleep resolves that.

This was an initial conclusion, because any type of change to the last command 'ps' allowed to overcome the issue and pass the test.